src/framework/mpe/soundid.h

```diff
@@ -193,11 +193,13 @@ enum class SoundId
     Tablas,
     Timbales,
     Anvil,
+    BellTree,
```

Comment:
You will also need to add these below in `static const std::unordered_map<SoundId, String> ID_STRINGS`.